### PR TITLE
fix(nginx): access log to tmpfs

### PR DESCRIPTION
**Pull request by william-gr**
Ticket:	#29361
(cherry picked from commit 5aa04b32af2872ccf9c772a04753aaf83daa0e2c)

### CODE REVIEW

File: src/freenas/usr/local/etc/syslog-ng.conf.freenas [hunk 1]
```diff
@@ -188,3 +193,11 @@ log { source(src); filter(f_emerg); destination(allusers); };
...
 log { source(src); filter(f_ppp); destination(ppp); };
 log { source(src); filter(f_mdnsresponder); filter(f_info); destination(mdnsresponder); };
+
+
+#
+# nginx.err                                             /var/log/nginx/error.log
+# nginx.info                                            /tmp/nginx-access.log
+#
+log { source(src); filter(f_nginx); filter(f_is_info); destination(nginx_access); };
+log { source(src); filter(f_nginx); filter(f_err); destination(nginx_error); };
```
Comment (darkfiberiru) [inline src/freenas/usr/local/etc/syslog-ng.conf.freenas]: Don't we want to add an f_is_err filter and use it here so we don't mix accesslogs into error logs?

Comment (darkfiberiru) [reply src/freenas/usr/local/etc/syslog-ng.conf.freenas]: @william-gr

Comment (william-gr) [reply src/freenas/usr/local/etc/syslog-ng.conf.freenas]: @darkfiberiru how could it happen if access log is only INFO?

Comment (darkfiberiru) [reply src/freenas/usr/local/etc/syslog-ng.conf.freenas]: Read man page backwards on log levels. All is good

**Pull request merged**